src/components/service_base.cpp

```diff
@@ -1 +1,4 @@
 #include <components/service_base.hpp>
+
+namespace viam {
+namespace cppsdk {
```

Comment:
Missing closing namespace `}`'s here. Is this file actually registered in the build system?

Reply:
Thanks for the catch, fixed. It looks like we have the header in the `CmakeLists.txt` but not the implementation file (which is probably fine since it's empty). Would rather not touch in this PR, but I actually already made a ticket to look into this file (which I suspect we can get rid of entirely with some light refactoring) [here](https://viam.atlassian.net/browse/RSDK-2718).